app/src/main/java/sdp/moneyrun/map/MapActivity.java

```diff
@@ -233,6 +234,8 @@ public void removeCoin(Coin coin) {
         }
         remainingCoins.remove(coin);
         collectedCoins.add(coin);
+        currentScore += coin.getValue();
+        currentScoreView.setText("Score: "+Integer.toString(currentScore));
```

Comment:
It would be better to use a resource string.

Reply:
Can we use a resource string for something that changes ?  the textview  is changed every time the player grabs a coin

Reply:
You could define a new string resource like this

Reply:
Ok I have changed the string resource